.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish container
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    name: Build and push Spilo container
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+      packages: 'write'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v4
+      with:
+        # list of Docker images to use as base name for tags
+        images: |
+          ghcr.io/zalando/spilo-14
+        # generate Docker tags based on the following events/attributes
+        tags: |
+          type=ref,event=tag
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to GHCR
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        context: "postgres-appliance/"
+        file: "postgres-appliance/Dockerfile"
+        push: true
```

Comment:
Before pushing we would like to test at least amd64 image, like it is done here: https://github.com/zalando/spilo/blob/71491609ae88180655afdbb76fb8692f612473b7/.github/workflows/tests.yaml#L29-L30

Reply:
I'll try adding that. Would need to test the action with a delayed push.

Reply:
Any news here, @DavidSpek?